mw-webapp/src/component/table/renderCellValue/renderCellSpan.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from "src/component/table/columns.module.scss";
+
+const renderCellSpanWithValue = (value: string, isDone?: boolean) => (
+  <span className={isDone ? styles.completed : styles.notCompleted}>
+    {value}
+  </span>
+);
+
+const renderCellEmptySpan = () => (
+  <span className={styles.notCompleted}>
+    &#8203;
+  </span>
+);
+
+/**
+ * Render cell's span
+ * @param {string} value
+ * @param {boolean} isDone
+ * @returns {JSX.Element} JSX.Element
+ */
+export const renderCellSpan = (value: string, isDone?: boolean): JSX.Element => (
+  (value !== "" && value !== " ") ?
+    renderCellSpanWithValue(value, isDone)
```

Comment:
Use polymorphism to avoid this strange logic everywhere where you will call this function